{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "       KeyError: 'pytestbdd_then_the records are as follows \"{}\"'\n       _pytest.fixtures.FixtureLookupError: ('pytestbdd_then_the records are as follows \"{}\"', <FixtureRequest for <Function test_on_a_rest_db>>)\n           pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: Then \"the records are as follows \"{}\"\". Line 10 in scenario \"On a rest db\" in the feature \"/home/sflorescu/Projects/end-presentation-/tests/py_test/ch8/features/get_scenario.feature\"\n", "test_name": "test_on_a_rest_db", "rerun": "0"}}, "suite_name": "test_some.py"}}}, "date": "January 24, 2022", "start_time": 1642987314.6337678, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "1", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "1"}